resources/js/screens/failedJobs/job.vue

```diff
@@ -0,0 +1,228 @@
+<script type="text/ecmascript-6">
+    import phpunserialize from 'phpunserialize'
+
+    export default {
+        components: {
+            'stack-trace': require('./../../components/Stacktrace').default
+        },
+
+        /**
+         * The component's data.
+         */
+        data() {
+            return {
+                ready: false,
+                retrying: false,
+                job: {}
+            };
+        },
+
+
+        /**
+         * Prepare the component.
+         */
+        mounted() {
+            this.loadFailedJob(this.$route.params.jobId);
+
+            document.title = "Horizon - Failed Jobs";
+
+            this.interval = setInterval(() => {
+                this.reloadRetries();
+            }, 3000);
+        },
+
+
+        /**
+         * Clean after the component is destroyed.
+         */
+        destroyed() {
+            clearInterval(this.interval);
+        },
+
+
+        methods: {
+            loadFailedJob(id) {
+                this.ready = false;
+
+                this.$http.get('/' + Horizon.path + '/api/jobs/failed/' + id)
+                    .then(response => {
+                        this.job = response.data;
+
+                        this.ready = true;
+                    });
+            },
+
+
+            /**
+             * Reload the job retries.
+             */
+            reloadRetries() {
+                this.$http.get('/' + Horizon.path + '/api/jobs/failed/' + this.$route.params.jobId)
+                    .then(response => {
+                        this.job.retried_by = response.data.retried_by;
+
+                    });
+            },
+
+
+            /**
+             * Retry the given failed job.
+             */
+            retry(id) {
+                if (this.retrying) {
+                    return;
+                }
+
+                this.retrying = true;
+
+                this.$http.post('/' + Horizon.path + '/api/jobs/retry/' + id)
+                    .then(() => {
+                        setTimeout(() => {
+                            this.reloadRetries();
+
+                            this.retrying = false;
+                        }, 3000);
+                    });
+            },
+
+
+            /**
+             * Convert exception to a more readable format.
+             */
+            prettyPrintException(exception) {
+                var lines = _.split(exception, "\n"),
+                    output = '';
+
+                lines.forEach(line => {
+                    output += '<span>' + line + '</span>';
+                });
+
+                return output;
```

Comment:
Since Lodash is used a lot, isn't it neater to use something like _.reduce?

```js
var lines = _.split(exception, "\n");

return _.reduce(lines, (output, line) => {
  return output += '<span>' + line + '</span>';
}, '');
```